internal/internal_task_handlers.go

```diff
@@ -186,9 +186,9 @@ func isDecisionEvent(eventType s.EventType) bool {
 		s.EventTypeWorkflowExecutionCanceled,
 		s.EventTypeWorkflowExecutionContinuedAsNew,
 		s.EventTypeActivityTaskScheduled,
-		s.EventTypeActivityTaskCancelRequested,
+		s.EventTypeActivityTaskCancelRequested, s.EventTypeRequestCancelActivityTaskFailed,
```

Comment:
Why is it on the same line?

Reply:
because they are for the same decision.

Reply:
Just realize that for cancel activity, the respond decision event will always be EventTypeActivityTaskCancelRequested.